### ISSUE TITLE
Responsive image jump

### ISSUE THREAD

**Issue by hongaar** [nabble/unveil2#18]
When i.e. Bootstraps `img-reponsive` class is used, a image jump can be seen after loading the placeholder image, and after loading the final image.

Related: https://github.com/nabble/unveil2/issues/15#issuecomment-221809002

**Comment by BeatLaG** [comment]
Hey Joram! Some news about this issue?

**Comment by hongaar** [comment]
Ideally, placeholder images and final images could be pre-loaded into memory first ([using `new Image()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image)), this would still cause a jump when initially loading the placeholder image though. This would need some code updates.

However, I'm afraid this won't be fixed soon, as this is not truly a critical issue, and not one that unveil is the root cause of anyway. It's a Bootstrap issue at heart, [see example](http://jsfiddle.net/1mc2cp06/1/).

**Comment by BeatLaG** [comment]
You are totally right about that in essence is a BS issue more than a plugin´s. But your tip I think that it is gonna be really interesting about walking that side.

Thanks a lot for your help once again :)
